rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -60,10 +60,9 @@ header {
 }
 
 #displayAddress {
+  > div:first-child {
+    margin-top: 30px;
+  }
   font-weight: bold;
   text-shadow: 2px 2px 10px white;
```

Comment:
I prefer to put the styles for the particular selector _first_ and any rules for the children after, like [this](https://github.com/rosadiaz/snow_queen_v2/pull/1#discussion_r238080020)

Reply:
done